### PR TITLE
Update deep edit tranforms

### PR DESCRIPTION
**Pull request by diazandr3s**
Signed-off-by: Andres <diazandr3s@gmail.com>

- Change transform to verify multimodality
- Delete spacing transform - It impacts inference time a lot and it doesn't increase performance

### CODE REVIEW

File: sample-apps/deepedit/lib/train.py [hunk 1]
```diff
@@ -129,7 +128,8 @@ def val_pre_transforms(self, context: Context):
             LoadImaged(keys=("image", "label"), reader="nibabelreader"),
             SingleLabelSingleModalityd(keys=("image", "label")),
             AddChanneld(keys=("image", "label")),
-            Spacingd(keys=["image", "label"], pixdim=self.target_spacing, mode=("bilinear", "nearest")),
+            # Spacing might not be needed as resize transform is used later.
+            # Spacingd(keys=["image", "label"], pixdim=self.target_spacing, mode=("bilinear", "nearest")),
```
Comment (aihsani) [inline sample-apps/deepedit/lib/train.py:132]: why is this added and commented instead of just being removed?

File: sample-apps/deepedit/lib/train.py [hunk 2]
```diff
@@ -95,9 +94,9 @@ def train_pre_transforms(self, context: Context):
         return [
             LoadImaged(keys=("image", "label"), reader="nibabelreader"),
             SingleLabelSingleModalityd(keys=("image", "label")),
-            # RandZoomd(keys=("image", "label"), prob=0.4, min_zoom=0.3, max_zoom=1.9, mode=("bilinear", "nearest")),
             AddChanneld(keys=("image", "label")),
-            Spacingd(keys=["image", "label"], pixdim=self.target_spacing, mode=("bilinear", "nearest")),
+            # Spacing might not be needed as resize transform is used later.
```
Comment (aihsani) [inline sample-apps/deepedit/lib/train.py:98]: why is this added and commented instead of just being removed?

**Review by SachidanandAlle — APPROVED** [review]

**Review by aihsani — CHANGES_REQUESTED** [review]

File: sample-apps/deepedit/lib/train.py (hunk 1 above)
Comment (diazandr3s) [reply sample-apps/deepedit/lib/train.py:132]: As a reference... just in case a researcher wants to use this transforms
